tests/Backend/Export/ExportSimpleTest.php

```diff
@@ -0,0 +1,168 @@
+<?php
+namespace Keboola\Test\Backend\Export;
+
+use Keboola\StorageApi\Downloader\BlobClientFactory;
+use Keboola\Test\StorageApiTestCase;
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\Client;
+use Symfony\Component\Filesystem\Filesystem;
+
+class ExportSimpleTest extends StorageApiTestCase
+{
+    /**
+     * @return void
+     */
+    public function setUp()
+    {
+        parent::setUp();
+        $this->initEmptyTestBucketsForParallelTests();
+    }
+
+    /**
+     * Copy of ExportParamsTest::testTableExportAsyncSliced with simple filters.
+     * @see \Keboola\Test\Backend\Export\ExportParamsTest::testTableExportAsyncSliced
+     *
+     * @dataProvider tableExportSimpleProvider
+     *
+     * @param int $expectedColumnsCount
+     * @param array[] $ignoreColumnKeys
+     * @return void
+     */
+    public function testTableExportAsyncSliced(array $exportOptions, array $expectedResult, $expectedColumnsCount, array $ignoreColumnKeys)
+    {
+        $importFile = __DIR__ . '/../../_data/users.csv';
+        $csvFile = new CsvFile($importFile);
+        $tableId = $this->_client->createTableAsync($this->getTestBucketId(self::STAGE_IN), 'users', $csvFile, array(
+            'columns' => $csvFile->getHeader(),
+        ));
+
+        $results = $this->_client->exportTableAsync($tableId, array_merge($exportOptions, array(
+            'format' => 'rfc',
+        )));
+
+        $exportedFile = $this->_client->getFile($results['file']['id'], (new \Keboola\StorageApi\Options\GetFileOptions())->setFederationToken(true));
+
+        $this->assertTrue($exportedFile['isSliced']);
+        $this->assertGreaterThan(0, $exportedFile['sizeBytes']);
+
+        $tmpDestinationFolder = __DIR__ . '/../_tmp/slicedUpload/';
```

Comment:
máš nějaký důvod proč dělat dočasnů složku tady a ne v tmp?

Reply:
ne, je to zkopirovany kod...

Reply:
OK, nahradil jsem pomoci `sys_get_temp_dir()`